spec/capybara_helper.rb

```diff
@@ -11,17 +13,28 @@
   Capybara.javascript_driver = :selenium
 else
   require 'capybara/poltergeist'
-  Capybara.register_driver :poltergeist do |app|
-    options = {
-      js_errors: true,
-      timeout: 120,
-      debug: false,
-      inspector: true,
-      ignore_ssl_errors: true
-    }
-    Capybara::Poltergeist::Driver.new(app, options)
+  Capybara.configure do |config|
+    config.default_driver = :poltergeist
+    config.javascript_driver = :poltergeist
+    config.current_driver = :poltergeist
+    config.run_server = true
+    config.default_wait_time = 10
   end
 
+  Capybara.register_driver :poltergeist do |app|
+  Capybara::Poltergeist::Driver.new(app,
+      :phantomjs_options => [
+          '--debug=no',
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.